src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareUtils.cs

```diff
@@ -28,6 +28,9 @@ internal static DacDeployOptions CreateSchemaCompareOptions(DeploymentOptions de
             PropertyInfo[] deploymentOptionsProperties = deploymentOptions.GetType().GetProperties();
 
             DacDeployOptions dacOptions = new DacDeployOptions();
+            var optionsMapTable = new Dictionary<string, DeploymentOptionProperty<bool>>();
+
+            // Get the optionsMapTable property which has the updated option values
             foreach (var deployOptionsProp in deploymentOptionsProperties)
```

Comment:
it looks like this look only does something if the deployOptionsProp is "ExcludeObjectTypes" or "OptionsMapTable". Does this need to loop through all the other options or can there just be logic for finding these two specific options and doing the actions needed for those two?

Reply:
In this block, the first foreach loop sets the ExcludeObjectTypes values to the property and fetches the OptionsMapTable and iterate through the boolean type properties inside OptionsMapTable. Since all other properties are not using in the ADS and they can be takes as default values. Thanks